src/components/Server.vue

```diff
@@ -86,13 +86,18 @@ export default defineComponent({
 .server {
   margin-bottom: 14px;
 
+  &__title:hover {
+     text-shadow: 0 0 2px var(--color-box-shadow-white);
```

Comment:
тут не нужна отдельная переменная, тк этот цвет зависит от цвета текста

```suggestion
     text-shadow: 0 0 2px color-mod(var(--color-text-main) alpha(30%))
```